docs/source/user/file_formats.rst

```diff
@@ -129,6 +127,24 @@ browser and select the “Editor” item in the “Open With” submenu:
     <iframe src="https://www.youtube-nocookie.com/embed/b5oAoVB3Wd4?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
   </div>
 
+JupyterLab's powerful grid viewer can load files up to the maximum string
+size in the browser. Our experience testing the grid viewer on various browsers suggests that maximum string size varies with browser.
+Below is a table that reflects the maximum size we've successfully tested the grid viewer by browser:
+
++---------+----------+
+| Browser | Max Size |
++=========+==========+
+| Firefox | ~200MB   |
++---------+----------+
+| Chrome  | ~700MB   |
++---------+----------+
+| Safari  | ~1.5GB   |
++---------+----------+
+| IE      | ~1.5GB   |
++---------+----------+
+
+These numbers are intented to be conservative estimates.  Additionally, string encoding will also affect the ability to load CSV-like files on JupyterLab.
```

Comment:
"These numbers are conservative estimates. The actual sizes of files that can be successfully loaded vary depending on the browser version and file content."

Reply:
Kept first sentence and changed second to: "The actual maximum size of files that can be successfully loaded will vary depending on the browser version and file content."